src/mrinufft/trajectories/display.py

```diff
@@ -58,33 +59,33 @@ class displayConfig:
     slewrate_point_color: str = "b"
     """Matplotlib color for slew rate constraint points, by default ``"b"`` (blue)."""
 
-    def __init__(self, **kwargs):
+    def __init__(self, **kwargs: Any) -> None:  # noqa ANN401
         """Update the display configuration."""
         self.update(**kwargs)
 
-    def update(self, **kwargs):
+    def update(self, **kwargs: Any) -> None:  # noqa ANN401
         """Update the display configuration."""
         self._old_values = {}
         for key, value in kwargs.items():
             self._old_values[key] = getattr(displayConfig, key)
             setattr(displayConfig, key, value)
 
-    def reset(self):
+    def reset(self) -> None:
         """Restore the display configuration."""
         for key, value in self._old_values.items():
             setattr(displayConfig, key, value)
         delattr(self, "_old_values")
 
-    def __enter__(self):
+    def __enter__(self) -> "displayConfig":
```

Comment:
if you use `from __future__ import annotations` you should be able to annotate with the Class directly, not a string :)

Reply:
Delayed annotation checks helped in several cases actually, thx !